madmom/ml/nn/layers.py

```diff
@@ -344,6 +344,144 @@ def activate(self, data):
         return out
 
 
+class GRUCell(object):
+    """
+    Cell as used by GRU layers proposed in [1].
+
+    Parameters
+    ----------
+    weights_in_to_hidden_update : numpy array, shape ()
+        Weights.
+    weights_in_to_hidden_update : numpy array, shape ()
+        Weights.
+    bias : scalar or numpy array, shape ()
```

Comment:
Again, I think `recurrent_weights` is more appropriate, since it is used as recurrent weights connecting to the previous time step if I'm not wrong.